src/view/styles/borderstyles.js

```diff
@@ -0,0 +1,211 @@
+/**
```

Comment:
This is awfully long :D But I realised that the only thing I care about is – what will happen with a style value which is not recognized by a processor? E.g., if they'll introduce a more complex `border` notation or some other ways to write a color or a possibilit to set multiple borders at once (I think it's even possible), then what will happen?

The thing is – those things must be preserved.